lib/inline_svg/transform_pipeline.rb

```diff
@@ -4,7 +4,7 @@ def self.generate_html_from(svg_file, transform_params)
       document = Nokogiri::XML::Document.parse(svg_file)
       Transformations.lookup(transform_params).reduce(document) do |doc, transformer|
         transformer.transform(doc)
-      end.to_html
+      end.to_html.strip!
```

Comment:
Do we want `strip!` or `strip` here? Seems like [`strip!` modifies the string in place, returning `nil` if no modifications are made](https://ruby-doc.org/3.2.2/String.html#method-i-strip-21). Maybe `strip` is more suitable here because it "[returns] a copy of the receiver with leading and trailing whitespace removed" and would never return `nil`. What do you think? 🤔